lib/scala/project-manager/src/main/resources/application.conf

```diff
@@ -51,7 +54,7 @@ project-manager {
 
   storage {
     projects-root = ${user.home}/enso
-    projects-root=${?PROJECTS_ROOT}
+    projects-root = ${?PROJECTS_ROOT}
```

Comment:
I cannot claim to be an expert on Scala and its `.conf` file. However I believe that the `${?SERVER_HOST}` definition reads value from environment property, if it is set.

I am surprised we need _so little changes_ in the `project-manager` code!